scripts/vm/hypervisor/xenserver/xenserver84/vmopsSnapshot

```diff
@@ -0,0 +1,622 @@
+#!/usr/bin/python
```

Comment:
python3 too ?

Reply:
btw: is there a script named `vmopspremium` ?

Reply:
i thought i changed this and there is a file names vmopspremium

Reply:
my bad - seems like I changed it on the env I was testing on :facepalm: